pom.xml

```diff
@@ -466,11 +484,11 @@
           <configLocation>google_checks.xml</configLocation>
           <linkXRef>false</linkXRef>
           <violationSeverity>warning</violationSeverity>
-          <includeTestSourceDirectory>true</includeTestSourceDirectory>
+          <includeTestSourceDirectory>false</includeTestSourceDirectory>
```

Comment:
Why don't we want to include the test code in checkstyle?

Reply:
I think it's to deliver tests faster so they can serve their purpose - to verify the code. Do you think it's critical to include them back?

Reply:
I think it is important to have consistent code, both test and src. Do you feel that this slows down test creation? I'm not sure why we'd need different formatting for tests. Perhaps the problem is that you don't have the checkstyle plugin in your IDE and you are finding out style differences at build time instead of during development?

Reply:
Sure, I have added tests back to checkstyle